scripts/prepare-release

```diff
@@ -44,9 +44,8 @@ fi
 # Create a new branch that bases off of current origin/master.
 git fetch origin master
 git switch -C "topic/release-${new_version}" origin/master
-
 # Update the fallback version and its ancestor count.
-rev_count="$(($(git rev-list --count) + 1))"
```

Comment:
@tobim please ack this change: What we had before did not work.

Reply:
Talked about this in a huddle; rev-count is a very confusing name, as this is actually ancestor-count + 1. I fixed the script and updated the comment to reflect that.

Reply:
We resolved it by passing `HEAD` to the command.